tikv-configuration-file.md

```diff
@@ -1163,6 +1163,16 @@ Configuration items related to `import`
 + The number of jobs imported concurrently
 + Default value: `8`
 + Minimum value: `1`
+  
+## backup
+
+Configuration items related to BR backup
+
+### `num-threads`
+
++ The number of worker threads to process backup
++ Default value: `CPU * 0.75`, but the maximum is `32`.
```

Comment:
if we follow [the English example of `max-thread-count`](https://github.com/pingcap/docs/blob/5da68fa907a4f57c035af8b2f153e476df5d57bc/tikv-configuration-file.md#max-thread-count) this should be written as 

```suggestion
+ Default value: `MIN(CPU * 0.75, 32)`.
```

is there any plan to make these expressions the same between the two languages :thinking: